docs/operator.md

```diff
@@ -1350,7 +1350,7 @@ file contains the following configuration options for the regular Percona XtraDB
 | **Key**         | {{ optionlink('backup.backoffLimit') }} |
 | **Value**       | int |
 | **Example**     | `6` |
```

Comment:
Not sure if you are putting defaults into these examples or not, so maybe change it to 10 here as well.